app/editor/src/components/tool-bar/ToolBarSection2.tsx

```diff
@@ -0,0 +1,21 @@
+import { Row } from 'tno-core';
+import * as styled from './styled';
+
+export interface IToolBarSection2Props {
+  label: string;
+}
+/**
+ * Creates a section for the parent ToolBar component
+ * @param actions Provide buttons/ filter actions for given section
+ * @param label Provide a label for the section
+ * @returns A ToolBar section
+ */
+export const ToolBarSection2: React.FC<IToolBarSection2Props> = ({ label }) => {
+  return (
+    <styled.ToolBarSection className="details-section">
+      <Row className="title-container">Content Details</Row>
```

Comment:
Could maybe make this a prop (`title: string`) so we can reuse this section for other areas!